ProcessMaker/Managers/ScreenBuilderManager.php

```diff
@@ -45,4 +39,27 @@ public function getScripts()
     {
         return $this->javascriptRegistry;
     }
-}
\ No newline at end of file
+
+    public function addPackageScripts($type)
+    {
+        // Depending of the form type we load the correct file with the components to display
+        switch ($type) {
```

Comment:
This logic could be clearer. We're adding a table for screen_types, can we use that instead?

Reply:
The code was simplified with the convention that the files with the extension components use the screen type (form, display, etc.) in the file name.